nav2_costmap_2d/include/nav2_costmap_2d/collision_base.hpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2020 Shivang Patel
```

Comment:
Please keep the names of people that had derived work included here too. I'd actually argue for this case since its mostly a refactor, your name should not be included in this paticular set of files.

Reply:
Sorry about that. I was actually going to add `modified by` section at the end.